R/tm_missing_data.R

```diff
@@ -157,9 +159,11 @@ tm_missing_data <- function(label = "Missing data",
   assert_decorators(decorators, null.ok = TRUE, names = available_decorators)
   # End of assertions
 
+  datanames <- unique(datanames)
   ans <- module(
     label,
     server = srv_page_missing_data,
+    datanames = datanames,
```

Comment:
Hmmm, `parent_dataname` seems to be relevant here. What would happen if ADSL is not in data
```suggestion
    datanames = union(datanames, parent_dataname), 
```